src/main/kotlin/racingcar/Cars.kt

```diff
@@ -0,0 +1,18 @@
+package racingcar
+
+import racingcar.condition.MoveCondition
+
+class Cars(carNumber: Int, conditions: List<MoveCondition>) {
+    private val cars: List<Car> =
+        IntRange(0, carNumber - 1).map { num ->
+            Car("${num}번 차", START_POSITION, conditions)
+        }
```

Comment:
```suggestion
        private val cars: List<Car> = List(carNumber) {
        Car("${it}번 차", START_POSITION, conditions)
    }
```
List()를 활용해 볼 수 있어요 :)

Reply:
List() 좋네요 👍